packages/core/core.test.mts

```diff
@@ -381,23 +381,6 @@ describe('Error handling', () => {
     await expect(answer).rejects.toThrowError('Error in render function');
   });
 
-  it('Prevent trying to run 2 prompts at once.', async () => {
```

Comment:
Side effect, it becomes theoretically possible to run 2 inquirer prompts at once.

This is unlikely to have much benefits but:
1. Could be theoretically possible to run prompts on different streams (if Node Readline allow it - it used to have singleton conflicts)
2. When running tests today, if a test fail Inquirer doesn't cleanup. This won't happen anymore 🎉